Trustroots/trustroots
# Homepage: accommodate cyclists circle on landing page

#### Proposed Changes

* Add cyclists board pic, and show it when landing page URL has `?circle=cyclists`
* Ensure two "join" CTAs on homepage link to signup form with circle slug
* Rename some "tribe" variables to "circles" while we're at it

#### Testing Instructions

* Have a circle with slug "cyclists" in your DB

* Open homepage as unlogged — you should see random photo on each load

* Open `?circle=cyclists` or `?tribe=cyclists` — you should always see cyclists photo
    <img width="1190" alt="Screenshot 2021-02-10 at 10 32 34" src="https://user-images.githubusercontent.com/87168/107490832-1153c800-6b93-11eb-863f-89ab493c24ec.png">

* You should also see cyclists circle lower on the page in "bubbles":
    <img width="1328" alt="Screenshot 2021-02-10 at 11 28 31" src="https://user-images.githubusercontent.com/87168/107490950-37796800-6b93-11eb-8789-4054353fb8b2.png">

* The two "join" CTAs will link to signup form with `?tribe=cyclists` URL param, and without it when no circle defined.
    <img width="665" alt="Screenshot 2021-02-10 at 11 24 58" src="https://user-images.githubusercontent.com/87168/107491057-57109080-6b93-11eb-8587-bd48b4ff648d.png">

* The header button won't do that, it's a bit more complicated so I wanted to have it in different PR, this was enough unfocused PR already... 😅 
    <img width="818" alt="Screenshot 2021-02-10 at 11 25 44" src="https://user-images.githubusercontent.com/87168/107492113-b4f1a800-6b94-11eb-9586-16e32bcccba6.png">
